### PR TITLE
option to initialize passive set P

### PR DESCRIPTION
**Pull request by JeffFessler**
addresses #17

### CODE REVIEW

**Comment by JeffFessler** [comment]
@lstagner, github recommended you for a code review.  you want to take a look or should i just proceed?

File: src/pivot_comb.jl [hunk 1]
```diff
@@ -1,23 +1,28 @@
...
-               B::AbstractMatrix{T};
-               tol::Float64=1e-8,
-               max_iter=30*size(A,2)) where {T}
+function pivot_comb(
+    A,
+    B::AbstractMatrix{T};
+    tol::Float64=1e-8,
+    max_iter=30*size(A,2),
+    P!::AbstractMatrix{Bool} = falses(size(A,2), size(B,2)),
+) where {T}
```
Comment (MatFi) [inline src/pivot_comb.jl:26]: Not sure, but do you think that we should test for `size(P!) == (size(A,2),size(B,2))`, and throw an error otherwise?

**Review by MatFi — COMMENTED** [review]
Looks good to me, optionally add a test with the case that `P!` is mutated.

File: src/pivot_comb.jl (hunk 1 above)
Comment (MatFi) [reply src/pivot_comb.jl:26]: Forget about it, because than we should also check the ’A’ and ’b’s and this is done nowhere ...

**Comment by JeffFessler** [comment]
@MatFi, I added a little test for `P!` and it works.  See if you think it's ready now?

**Review by MatFi — APPROVED** [review]

**Pull request merged**